Source/SteveSloth/Private/MyPermanentHealthPickup.cpp

```diff
@@ -0,0 +1,31 @@
+/****************************************************************************************
+ * Copyright:
+ * Name: Guy Lapensee
+ * Script: AMyPermanentHealthPickup
+ * Date: May 16, 2024
+ * Description:Leaf  pickup for 25 permanent health boost
+ * TODO: Permanently add health to the players health bar
+ * Known Bugs:
+ ****************************************************************************************/
+
+#include "MyPermanentHealthPickup.h"
+
+AMyPermanentHealthPickup::AMyPermanentHealthPickup()
+{
+	ItemClass = CreateDefaultSubobject<AItemBaseClass>(TEXT("Item Class"));
+	PlayerClass = CreateDefaultSubobject<AMyPlayer>(TEXT("Player Class"));
```

Comment:
You dont access these this way, this is setting them as a component, can be removed